lib/Unicode/Char/Numeric.hs

```diff
@@ -41,3 +45,15 @@ isNumber c = case generalCategory c of
     LetterNumber  -> True
     OtherNumber   -> True
     _             -> False
+
+
+-- | Numeric value of a character, if relevant.
+{-# INLINE numericValue #-}
+numericValue :: Char -> Maybe Rational
+numericValue = V.numericValue
+
+-- | Integer value of a character, if relevant.
+intValue :: Char -> Maybe Int
+intValue c = V.numericValue c >>= \r -> if denominator r == 1
+    then Just (fromIntegral (numerator r))
+    else Nothing
```

Comment:
Generally the last line in a file ends with a newline, github warns about it.

Reply:
Yes. I updated the `.editorconfig` file.